controllers/apps/transformer_cluster_deletion.go

```diff
@@ -166,33 +164,9 @@ func kindsForWipeOut() ([]client.ObjectList, []client.ObjectList) {
 
 // shouldSkipObjOwnedByComp is used to judge whether the object owned by component should be skipped when deleting the cluster
 func shouldSkipObjOwnedByComp(obj client.Object, cluster kbappsv1.Cluster) bool {
```

Comment:
remove `shouldSkipObjOwnedByComp`  and use `isOwnedByComp` directly

Reply:
done